stdlib.sh

```diff
@@ -486,7 +501,7 @@ use_nix() {
 
 ## Load the global ~/.direnvrc if present
 if [[ -f ${XDG_CONFIG_HOME:-$HOME/.config}/direnv/direnvrc ]]; then
-  source_env "${XDG_CONFIG_HOME:-$HOME/.config}/direnv/direnvrc" >&2
+  source_env ${XDG_CONFIG_HOME:-$HOME/.config}/direnv/direnvrc >&2
 elif [[ -f $HOME/.direnvrc ]]; then
-  source_env "$HOME/.direnvrc" >&2
+  source_env $HOME/.direnvrc >&2
```

Comment:
I'm not sure why the quotes were removed here. On OSX the home directly can be "/Users/Foo Bar" I think